docs/notebooks/notdiamond_custom_routing.ipynb

```diff
@@ -0,0 +1,476 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
```

Comment:
Note: this notebook won't show up in public docs until we autogen docs. Doc autogen is still a manual process as it is a bit fragile. We will have to run autogen after this merges

Reply:
Gotcha, thank you for clarifying this!